src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationInputObjectType.cs

```diff
@@ -13,7 +13,7 @@ public LocalizationInputObjectType(IStringLocalizer<LocalizationInputObjectType>
             Description = S["the localization part of the content item"];
 
             AddScalarFilterFields<StringGraphType>("culture", S["the culture of the content item to filter"]);
-            Field(x => x.LocalizationSet).Description(S["the localization set of the content item to filter"]);
+            Field(x => x.LocalizationSet, true).Description(S["the localization set of the content item to filter"]);
```

Comment:
can you use a named param? eg `nullable: true` - it helps someone reading the code to understand what the overload is doing,